wallet-new/server/Cardano/Wallet/Server/Plugins.hs

```diff
@@ -132,7 +133,8 @@ walletBackend (NewWalletBackendParams WalletBackendParams{..}) passive =
 
 -- | A @Plugin@ to resubmit pending transactions.
 resubmitterPlugin :: (HasConfigurations, HasCompileInfo) => Plugin WalletWebMode
-resubmitterPlugin = ([ActionSpec $ \diffusion -> startPendingTxsResubmitter (sendTx diffusion)], mempty)
+resubmitterPlugin = ([ActionSpec $ \diffusion -> askWalletDB >>= \db ->
+                         startPendingTxsResubmitter db (sendTx diffusion)], mempty)
```

Comment:
Yes, here `askWalletDB` yields the `acid-state` DB handle, so that's obviously necessary to pass it along and have the async worker re-read the snapshot every time, to be able to resubmit transactions.